dynawo/sources/Modeler/DataInterface/IIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -1030,6 +1030,20 @@ DataInterfaceIIDM::findCalculatedBusInterface(const string& voltageLevelId, cons
   throw DYNError(Error::MODELER, UnknownCalculatedBus, voltageLevelId, node);
 }
 
+shared_ptr<BusInterface>
+DataInterfaceIIDM::findCalculatedBusInterface(const string& voltageLevelId, const string& bbsId) {
+  map<string, vector<shared_ptr<CalculatedBusInterfaceIIDM> > >::iterator iter = calculatedBusComponents_.find(voltageLevelId);
+  if (iter == calculatedBusComponents_.end())
+    throw DYNError(Error::MODELER, UnknownVoltageLevel, voltageLevelId);
```

Comment:
same as above

Reply:
I'll reuse UnknownCalculatedBusWithBbs since there is a bbsId

Reply:
This function doesn't throw error any more.
An empty shared pointer is returned if no calculated bus is found.